### DISCUSSION TITLE
Strange issue with USB tethering

### DISCUSSION THREAD

**Discussion by Ichisuke83** [Mygod/VPNHotspot#484] [Q&A]
Hi there. I have this strange behaviour.
I'm using a Samsung smartphone with Lineage OS.
I set up a VPN for USA region and use VPNHotspot. I want to see some streaming content and I've found out my LG TV can use USB tethering!
The streaming app load correctly and I can see the menus on the screen, even previews of the movies on their pages, browse the catalog and whatever. But then the actual stream doesn't work. It continuously load and then throws a generic error.

Point is that if I use the VPN with mobile data, VPNHotspot and set up the WiFi hotspot I can also start the streaming correctly. But since my data plan is limited I'd like to use USB tethering since that uses the WiFi home connection.

Why is it doing that?

**Comment by Mygod** [comment]
Try reapply routing rules, restart phone, etc. 🤔

**Reply by Ichisuke83** [reply]
Already done that. Nothing changes. I think the streaming app for some reason detect "something" unusual and block the actual streaming of the movies but the menus/interface load fine. There's probably something going on with USB tethering/VPN with blocked content using the TV. The same setup with my computer works just fine. But on Windows the streaming is done via Chrome since there is no official app.

Edit: solved the issue by using another smartphone, an LG G5, with the repeater function (the old Samsung didn't have that), USB tethering was a no on the G5.
